### PR TITLE
Add ci build script to repo

### PR DESCRIPTION
**Pull request by amisevsk**
### What does this PR do?
Adds a ci script to `./.ci/` to simplify CI process

### CODE REVIEW

**Comment by amisevsk** [comment]
This should be the release CI script, afaik.

**Comment by riuvshin** [comment]
this is not the release CI script this is more like nightly one.

File: .ci/build.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+set -e
+
+TAG=$(echo $GIT_COMMIT | cut -c1-7)
+docker login -u ${DOCKER_HUB_LOGIN} -p ${DOCKER_HUB_PASSWORD}
+
+# Run tests and all the other checks on repo
+docker build -f build/CI/Dockerfile .
+
+# Unified Plugin Broker
+docker build -t eclipse/che-unified-plugin-broker:latest -f build/unified/Dockerfile .
```
Comment (riuvshin) [inline .ci/build.sh]: please fix ctx and paths

Comment (amisevsk) [reply .ci/build.sh]: Should be better now. Tested locally without docker login and push commands and it works.

**Pull request merged**